modules/claims_api/app/controllers/claims_api/v1/forms/disability_compensation_controller.rb

```diff
@@ -252,7 +252,9 @@ def valid_526_response
 
         def format_526_errors(errors)
           errors.map do |error|
-            { status: 422, detail: "#{error['key']} #{error['detail']}", source: error['key'] }
+            e = error.deep_symbolize_keys
+            details = e[:text].presence || e[:detail]
+            { status: 422, detail: "#{e[:key]}, #{details}", source: e[:key] }
           end
```

Comment:
@stiehlrod  I believe this change will also need to be applied in the `v1/forms/claims_controller` error formatter `format_evss_errors` method

Reply:
also, with this change in the validate endpoint we should be able to return to our previous state with this spec https://github.com/department-of-veterans-affairs/vets-api/pull/15437/files#diff-53e8c781b07d98261e04fbd91c9dd42787f68e6f06874b2f5b2e8912ddf527b3 .